common/djangoapps/student/helpers.py

```diff
@@ -325,7 +325,7 @@ def _get_redirect_to(request):
     # get information about a user on edx.org. In any such case drop the parameter.
     if redirect_to:
         mime_type, _ = mimetypes.guess_type(redirect_to, strict=False)
-        if not _is_safe_redirect(request, redirect_to):
+        if not is_safe_redirect(request, redirect_to):
```

Comment:
@nasthagiri: FYI: The caller of `is_safe_redirect` ([_get_redirect_to](https://github.com/edx/edx-platform/blob/33c5a73bd050804f7ce2eae54971968865c25e57/common/djangoapps/student/helpers.py#L311-L368)) adds additional checks that weren't in the original logout next check.  @douglashall voted that if we want to expand checks, that should come later.  I'm fine with that and creating another ticket.  You on board?